common/djangoapps/student/migrations/0022_auto_20190624_1901.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-06-24 19:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('student', '0021_historicalcourseenrollment'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
```

Comment:
FYI @edx/devops , this will be a pretty big migration, but my memory is that it should be fine on Aurora.